### ISSUE TITLE
Sharing is broken since 2023.1.21

### ISSUE THREAD

**Issue by friedrichkeydel** [marius-wieschollek/passwords#564]
### System Information
**Server:**
```json
{
    "version": {
        "server": "25.0.2.3",
        "app": "2023.1.21",
        "lsr": false,
        "php": "8.1.13",
        "cronPhp": "8.1.13"
    },
    "environment": {
        "os": "Linux",
        "architecture": "x86_64",
        "bits": 64,
        "database": "mysql",
        "cron": "cron",
        "proxy": false,
        "sslProxy": false,
        "subdirectory": false
    },
    "services": {
        "images": "gdlib",
        "favicons": "fg",
        "previews": "pageres",
        "security": "hibp",
        "words": "leipzig",
        "previewApi": false,
        "faviconApi": false
    },
    "status": {
        "autoBackupRestored": false
    },
    "settings": {
        "channel": "stable",
        "nightlies": false,
        "handbook": false,
        "performance": 5
    },
    "encryption": {
        "sse": {
            "SSEv1r1": false,
            "SSEv1r2": true,
            "SSEv2r1": false,
            "none": false,
            "default": "SSEv1r2"
        },
        "cse": {
            "CSEv1r1": false,
            "none": true,
            "default": "none"
        }
    }
}

 Nextcloud Version:
 PHP Version: 8.1
 Database and Version: MariaDB 10.5.18
 Server OS and Version: Ubuntu 20.04
 Passwords Version: 2023.1.21
```

**Client:**
```
 Browser and Version: Firefox 108.0.2
 Client OS and Version: MacOS 13.1
```

### Steps to reproduce
1. select password entry
2. click share icon
3. select a prtson to share with

### Expected result
What should happen?
Password entry should be shared to selected person

### Actual result
What does happen?
Loading animation circles forever

### Nextcloud log
<details>
<summary>Nextcloud log</summary>

```
 - Open the Nextcloud admin settings
 - Open the "Log" section
 - Click the "Copy" icon, then "Copy Raw"
```
</details>

### Browser log
<details>
<summary>Browser log</summary>

```
Press F12, copy the content of the console tab
```
</details>

**Comment by marius-wieschollek** [comment]
That is a PHP language level error. Are you using PHP 8.1 to run cronjobs?

**Comment by friedrichkeydel** [comment]
Sorry for that, I just ran the system report command with the wrong interpreter - I updated the output information above. The problem still persists.

**Comment by marius-wieschollek** [comment]
Could this be the same issue as #563 ? There is a `Object of class OC\\DB\\QueryBuilder\\QueryBuilder could not be converted to string` in the sharing task

**Comment by friedrichkeydel** [comment]
This could indeed be the case as I updated the module today and the error you mentioned has also been logged today for the first time

**Comment by dsofeir** [comment]
Hello, I have encountered the same issue using the Nextcloud 25 snap on Ubuntu 20.04

**Comment by ghost** [comment]
same problem on Nextcloud AIO stable.

**Comment by j0schi** [comment]
Same Problem for me, cant share PW`s anymore... Just wanted to let you know.

**Comment by marius-wieschollek** [comment]
Can you check which version you're using? If it's 2023.1.23 or 2023.1.13, please check your nextcloud log for any error messages.

**Comment by friedrichkeydel** [comment]
It seems to be fixed for me with Nextcloud Version 25.0.3 and Passwords Version 2023.1.23

**Comment by dsofeir** [comment]
Updated to 2023.1.13, sharing is working correctly now. Thank you
